sdk/src/types/block/semantic/mod.rs

```diff
@@ -111,6 +113,14 @@ impl<'a> SemanticValidationContext<'a> {
                 Output::Delegation(output) => (output.amount(), None, output.unlock_conditions()),
             };
 
+            if unlock_conditions.addresses().any(Address::is_implicit_account_creation) {
+                if has_implicit_account_creation_address {
+                    return Ok(Some(TransactionFailureReason::SemanticValidationFailed));
```

Comment:
This error here is because there can be only one implicit account creation input?

Reply:
Yes

Reply:
I really don't like this SemanticValidationFailed :angry:

Reply:
I have asked them to add a new variant btw, will change when they do